### ISSUE TITLE
Support for pyproject.toml?

### ISSUE THREAD

**Issue by woodruffw** [di/bump#22]
`bump` currently reads from a `.bump` file or from `setup.cfg` -- it would be cool if it could (additionally) read its settings from a `[tool.bump]` section within `pyproject.toml`!

It's my understanding that similar developer tools have done this on an opt-in basis with an extra, so `pip install 'bump[toml]'` could be used to enable this for interested users while leaving the default installation's behavior alone.

**Comment by woodruffw** [comment]
By way of example, this `.bump` file:

```ini
[bump]
input = foo/version.py
reset = true
```

would become this TOML section:

```toml
[tool.bump]
input = "foo/version.py"
reset = true
```

**Comment by di** [comment]
Makes sense to me! I don't think the extra is necessary.

**Comment by di** [comment]
Fixed in #25.
